helpers/filterSeats.js

```diff
@@ -60,19 +66,58 @@ module.exports = function(dust) {
       });
     }
 
+    // get the names of the best two sections from the theatre
+    function getBestSections(bandColours) {
+      var bestSections = [];
+      // add all section names to an array
+      _.forEach(bandColours, function(bands, section) {
+        bestSections.push(section);
+      });
+      // find the best section
+      bestSections = _.last(bestSections); // just Stalls
+      return bestSections;
+    }
+
+    // compare prices
+    function chosenForYou(ticketRate, ticketRateSection) {
+      var bestSections = getBestSections(params.bandColours);
+      var ticketRatePrice = ticketRates[ticketRate.ids].grossPrice;
+      // check if current ticketRate exists in
```

Comment:
If it just exists. The `chosenForYou` function gets a best value seat in the best section. So for instance if there is a gold seat that is cheaper than a silver seat in stalls, that seat will be great value and potentially given the `bestOffer` variable further down.